plugins/SimulationView/SimulationViewProxy.py

```diff
@@ -46,6 +46,14 @@ def currentLayer(self):
     def minimumLayer(self):
         return self._simulation_view.getMinimumLayer()
 
+    @pyqtProperty(float, notify=currentLayerChanged)
+    def currentLayerHeight(self):
+        return self._simulation_view.getCurrentLayerHeight()
+
+    @pyqtProperty(float, notify=currentLayerChanged)
+    def minimumLayerHeight(self):
+        return self._simulation_view.getMinimumLayerHeight()
```

Comment:
These will recalculate (go through the entire scene, for every layer) _each_ time, _every_ time you request an update.

Please cache this at the very least, but even better, just go through the layers once on receiving them, and then save all of that; then you can query that structure.